research-software-directory/RSD-as-a-service
# Allow adding community categories to software

## Allow adding community categories to software

Changes proposed in this pull request:

* When joining a community, you are prompted to add the corresponding categories to the software page
* After joining, you can still edit the categories

How to test:

* `docker compose down --volumes && docker compose build --parallel && docker compose up --scale data-generation=1`
* Sign in as admin
* Join some communities on some software pages
* Play around with adding and removing categories
* Play around with joining and leaving communities
* Check that on the software page, the categories are always correctly shown

**Note**: I'm open for stylistic changes, but changing the workflow would take too much effort/time in this PR, unless something is fundamentally broken. We want to show this to an external partner in a week.


Closes #1212

PR Checklist:
